standalone/build.gradle.kts

```diff
@@ -51,10 +47,6 @@ tasks.named<Test>("test") {
         jvmArgs("--add-exports", "java.base/sun.security.ec=ALL-UNNAMED")
     }
 
-    // Add wolfcrypt JNI lib path to LD_LIBRARY_PATH (as our native library loading does not handle it)
-    environment(
-            "LD_LIBRARY_PATH", "$rootDir/ext/wolfcrypt-jni/lib/:" + System.getenv("LD_LIBRARY_PATH")
-    )
```

Comment:
This  may still be necessary no? Because wolfCrypt has their own JNI lib and that needs to be on the LD_LIBRARY_PATH because their provider loads it. Or is this gone because the submodule will be removed and it will all go through a nix package?